app/styles/_variables.scss

```diff
@@ -345,7 +345,6 @@ $overlay-background-color: rgba(0, 0, 0, 0.4);
 
   /** Dialog */
   --dialog-warning-color: $yellow-600;
-  --dialog-warning-text-color: $yellow-800;
```

Comment:
This variable was not used anywhere so I dropped it. It's not necessary for this PR but it's such a minor change that I didn't quite have the energy to pull it out into a separate PR.